grpc-server/build.gradle

```diff
@@ -13,6 +13,6 @@ dependencies {
     compile "log4j:log4j:1.2.17"
 
     testCompile "io.grpc:grpc-testing:${revGrpc}"
-    testCompile 'org.testinfected.hamcrest-matchers:all-matchers:1.8'
+    testCompile "org.testinfected.hamcrest-matchers:all-matchers:${revHamcrestAllMatchers}"
```

Comment:
Can this version be updated in the `versionsOfDependencies.gradle`?

Reply:
I moved this version to `versionsOfDependencies.gradle` file, because I'm using it also in the postgres and mysql modules.
Unfortnuly 1.8 is the latest: [all-matchers](https://mvnrepository.com/artifact/org.testinfected.hamcrest-matchers/all-matchers)